<html>
  <head>
    <link rel="stylesheet" type="text/css" href='https://fonts.googleapis.com/css?family=Ubuntu+Mono' rel='stylesheet' type='text/css'>
    <style>
      * {
      font-family:  'Ubuntu Mono';
      <!-- font-size: 14px; -->
      }
    </style>
  </head>
  
  <body>
    <div class="post-entry">
      <p><a href="https://raw.githubusercontent.com/stephaneguindon/stephaneguindon.github.io/master/moi.jpg"><img class=" wp-image-1128 alignnone" src="https://raw.githubusercontent.com/stephaneguindon/stephaneguindon.github.io/master/moi.jpg" alt="" width="166" height="191" /></a></p>
      <p><span><br />
          <strong> Email</strong>: guindon@lirmm.fr<br />
          <strong> Address</strong>: LIRMM CNRS UMR 5506, 860 rue de St Priest
          34095 Montpellier cedex 5<br />
      </span></p>
      <p align="justify" style="width:70%"> I design algorithms and probabilistic models of
        molecular evolution. Most notably, I am developing the software
        package <a href="http://code.google.com/p/phyml/">PhyML</a> (for Phylogenetics through
        Maximum Likelihood) which serves as a basis to implement my research outputs. I was trained
        as a biologist/statistician but I am now working in the computer science department of the <a href="http://www.lirmm.fr">LIRMM</a>
        in Montpellier, France. I also worked for
        the <a href="http://www.stat.auckland.ac.nz">Department of Statistics</a> at the University
        of Auckland. I am currently Associate Editor for BMC Evolutionary Biology and an elected Council member of the Society of Systematic Biologists (2013-2016).
      </p>
      <p>&nbsp;</p>
      <p align="justify" style="width:70%"><strong><font color="#ff5c33">Please contact me by email
      for information about current postdoc opportunities (one open position as of February 2018).</font></strong></p>
      <p>&nbsp;</p>
      <p><strong>Recent publications</strong> (full list
      on <a href="http://scholar.google.com/citations?sortby=pubdate&amp;hl=en&amp;user=kvrvJccAAAAJ&amp;view_op=list_works">Google
      Scholar</a>)</p>


      <p align="justify" style="width:70%"> <em><u>Accounting for calibration uncertainty: Bayesian
      molecular dating as a ``doubly intractable'' problem.</em> S. Guindon. Systematic
      Biology. 67: 651-661. 2018.</u> I describe a technique for molecular dating that accomodates for
      uncertainty in the placement of calibration constraints as defined by an expert. The method
      relies on the so-called ``exchange algorithm'' for sampling from doubly intractable
        distributions <a href="https://academic.oup.com/sysbio/issue/67/4">Cover of July issue!</a>
        <br/>

      <p align="justify" style="width:70%"> <em><u>Demographic inference under the coalescent in a
        spatial continuum.</em> S. Guindon, H. Guo, D. Welch. Journal of Theoretical Population
        Biology. 111: 43â€“50. 2016.</u> We describe a method that fits a structured coalescent model
        assuming that individuals are scattered on a continuum rather that distributed in discrete
        demes. We show that  the density of the population and  the rate of dispersal of
        individuals can be inferred simultaneousy from the analysis of geo-referenced genetic
        sequence using this technique.<br/>

      <p align="justify" style="width:70%"> <em><u>Modeling competition and dispersal in a statistical phylogeographic
        framework.</em> L. Ranjard, D. Welch, M. Paturel, S. Guindon. Systematic
        Biology. 63:743-752. 2014</u>. We describe a model where the probability for a species to colonize an empty island
        at some point during the course of evolution is the same at
        that of an occupied one only if species do not compete with each other. Using simulations, we show that these probabilities can indeed be
        estimated from geo-referenced genetic sequences. <br/ >
        
      <p align="justify" style="width:70%"> <em><u>Performance of standard and stochastic branch-site models for detecting positive selection amongst coding sequences</em>
        A. Lu, S. Guindon. Molecular Biology and Evolution. 31: 484-495. 2014</u>. We
        describe the performance of the stochastic branch-site model (see S. Guindon et al. PNAS. 101:12957-12962. 2004.) in terms of type-I and power for detecting
        positive selection in coding sequences. Results indicate that this  approach
        is more suited than the standard branch-site model (<em>sensu</em> codeml)   in cases
        where it is not
        known <em>a priori</em> which lineages may have evolved under positive selection.<br />

      <p align="justify" style="width:70%"> <em><u>From trajectories to averages: an improved description of the heterogeneity of
      substitution rates along lineages</em>. S. Guindon. Systematic
      Biology. 62:22-34. 2013</u>. Assuming  that the evolution of the substitution rate at each position along a sequence is
      a realization of a (geometric) Brownian process,  the rate averaged
      over a given time interval is approximately gamma distributed. This study shows that ignoring the stochasticity of
      average substitution rates leads to poor estimates of important evolutionary parameters. The proposed
      approach also provides an efficient implementation of the
      covarion model that does not require augmentation of the state space. <br /> 


      <p>&nbsp;</p>
      <p><strong>Academic record &amp; appointments</strong></p>
      <table style="border:hidden;width:90%">
        <col style="width:8%">
        <col style="width:92%">
        <tbody>
          <tr>
            <td align="left" valign="top"><em>2006-...</em></td>
            <td align="left">CNRS researcher.</td>
          </tr>
          <tr>
            <td align="left" valign="top"><em>2007-2016</em></td>
            <td align="left"> Lecturer in the Department of Statistics, The University of Auckland.</td>
          </tr>
          <tr>
            <td align="left" valign="top"><em>2003-2005</em></td>
            <td align="left">Postdoc, School of Biological Sciences, The University of Auckland.<br />
              Supervisor: Allen Rodrigo<br />
              Title: <em> Molecular evolution of the HIV-1 genome </em></td>
          </tr>
          <tr>
            <td align="left" valign="top"><em>1999-2003</em></td>
            <td align="left">PhD student in Biology. LIRMM-CNRS, Montpellier.<br />
              Supervisor: Olivier Gascuel<br />
              Title: <em> Approche statistique pour la reconstruction de phylog&eacute;nies mol&eacute;culaires </em> (Statistical approach for building molecular phylogenies)<br />
              A copy of my thesis can be found <a title="These" href="http://www.lirmm.fr/~guindon/these_guindon.pdf">here</a>.</td>
          </tr>
          <tr>
            <td align="left" valign="top"><em>1998-1999</em></td>
            <td align="left">Master Thesis in Biology, Universit&eacute; Claude Bernard, Lyon.<br />
              Supervisors: Guy Perri&egrave;re and Manolo Gouy<br />
              Title: <em> Les transferts horizontaux de mat&eacute;riel g&eacute;n&eacute;tique chez les procaryotes </em> (Horizontal transfers between procaryote genomes</td>
          </tr>
        </tbody>
      </table>
      <p>&nbsp;</p>
      <p><strong>Students</strong></p>
      <table style="border:hidden;width:90%">
        <col style="width:8%">
        <col style="width:92%">
        <tbody>
          <tr>
            <td>2008</td>
            <td>Lin Ying Wee, PgDip, &#8220;<em>Hepatitis B virus evolution: positive selection and substitution rates</em>&#8221;</td>
          </tr>
          <tr>
            <td>2008</td>
            <td>Sandunie Dineika Chandrananda, BSc Hons, &#8220;<em>A Phylogenomic analysis of the Yeast genome</em>&#8221;</td>
          </tr>
          <tr>
            <td>2009</td>
            <td>Maha Ahmed Baker, 6 month internship, &#8220;<em>Graph algorithms applied to phylogenetics</em>&#8221;</td>
          </tr>
          <tr>
            <td>2010</td>
            <td>Lin Ying Wee, Masters, &#8220;<em>Detecting differences in the rates of evolution after gene duplication</em>&#8220;</td>
          </tr>
          <tr>
            <td>2010</td>
            <td>Naveen Joshi, 6 month internship, &#8220;<em>Deciphering the patterns of variations of evolutionary rates along yeast genomes</em>&#8221;</td>
          </tr>
          <tr>
            <td>2010</td>
            <td>Samuel Pichot, 6 month internship, &#8220;<em>Graph algorithms to untangle phylogenetic trees: application to phylogeography</em>&#8220;</td>
          </tr>
          <tr>
            <td>2011</td>
            <td>Eric Frichot, 6 month internship, &#8220;<em>Stochastic models of the evolution of protein lengths</em>&#8221;</td>
          </tr>
          <tr>
            <td>2011</td>
            <td>Yi Lu, Masters, &#8220;<em>Comparison of methods for detecting natural selection in coding sequences</em>&#8221;</td>
          </tr>
          <tr>
            <td>2011</td>
            <td>Louis Ranjard, postdoc, &#8220;<em>Developing new methods in statistical phylogeography</em>&#8221;</td>
          </tr>
          <tr>
            <td>2012</td>
            <td>Marie Paturel, 6 month internship, &#8220;<em>Testing new methods in statistical phylogeography</em>&#8221;</td>
          </tr>
          <tr>
            <td>2013</td>
            <td>Helen Shearman, PhD, &#8220;<em>Statistical methods for measuring biodiversity</em>&#8221;</td>
          </tr>
          <tr>
            <td>2013</td>
            <td>Serg Krasnozhon, PhD, &#8220;<em>Statistically-based graphical method for characterizing molecular evolutionary processes</em>&#8221;</td>
          </tr>
          <tr>
            <td>2013</td>
            <td>Spencer Enesa, Masters &#8220;<em>Connections between the coalescent and birth-death sampling processes</em>&#8221;</td>
          </tr>
          <tr>
            <td>2014</td>
            <td>Kelly Lu, Masters &#8220;<em>The coalescent with randomly fluctuating population size</em>&#8221;</td>
          </tr>
          <tr>
          <td>2014</td>
          <td>Hongbin Guo, BSC, Hons &#8220;<em>Extensions of Kingman&#8217;s coalescent</em>&#8221;</td>
          </tr>
          <tr>
          <td>2017</td>
          <td>Charles Dutertre, Masters &#8220;<em>Statistical expectations under the spatial
          &Lambda;-Fleming-Viot model</em>&#8221;</td>
          </tr>
          <tr>
          <td>2017</td>
          <td>Adrien Oliva, research engineer project &#8220;<em>Handling ambiguities in the
          reconstruction of ancestral sequences</em>&#8221;</td>
          </tr>
          <tr>
          <td>2018</td>
          <td>Marie Suez, postdoc project (funded by LabEX NUMEV) &#8220;<em>Graphical displays of
          mutation maps</em>&#8221;</td>
          </tr>
        </tbody>
      </table>
      <p>&nbsp;</p>
    </div><!-- end of .post-entry -->
    
  </body>               
</html>          

